proxy/src/main/java/com/velocitypowered/proxy/connection/backend/BackendPlaySessionHandler.java

```diff
@@ -103,8 +103,12 @@ public boolean handle(PluginMessage packet) {
     }
 
     if (PluginMessageUtil.isMcBrand(packet)) {
-      PluginMessage rewritten = PluginMessageUtil.rewriteMinecraftBrand(packet,
-          server.getVersion());
+      PluginMessage rewritten;
+      if (playerConnection.getProtocolVersion().compareTo(ProtocolVersion.MINECRAFT_1_8) >= 0) {
+        rewritten = PluginMessageUtil.rewriteMinecraftBrand(packet, server.getVersion());
```

Comment:
Could we just add a `ProtocolVersion` parameter for `rewriteMinecraftBrand` instead?